docs/admin-guide/loqusdb_integration.md

```diff
@@ -1,71 +1,64 @@
 
 
-### Configure One LoqusDB
-
-In the config `config.py` give the connection information like:
-
-```python
-#connection details for LoqusDB MongoDB database
-LOQUSDB_SETTINGS = dict(
-    binary_path='/path/to/bin/loqusdb',
-    config_path=<path/to/loqus/config>
-)
-```
-
-
+## Option 2: connecting to a loqussbapi instance via REST API
 
-### Configure Multiple LoqusDB
+Perhaps this option represents the easiest and the long-term supported option to establish a connection and run queries to loqusdb.
+Loqusdbapi should be running on a server with host and port reachable by the scout instance. Follow the instructions provided on the package [home page][loqussbapi] to either install it or run it in a Docker container.
 
-It is possible to configure LoqusDB per Institute.
+## Configuring the loqusdb connections in Scout using the config file
 
-1. Edit config.py
-To configure a separate LoqusDB give it reference "id". If needed configure an (optional) config_path. An entry with id "default" should always exist.
+Once a loqusdb instance is created via one one the two options (or both actually, since Scout supports the connection to multiple loqusdb instances), the `LOQUSDB_SETTINGS` parameter in the Scout config file (config.py) should be edited as shown in the following example:
 
 ```
-    LOQUSDB_SETTINGS = [
-        {"binary_path": "/path/to/bin/loqusdb", "id": "default", "config_path":"loqus.cfg"},
-        {"binary_path": "/path/to/bin/loqusdb", "id": "foo", "config_path":"bar.cfg"}
-    ]
+LOQUSDB_SETTINGS = {
+  "default" : {"binary_path": "/miniconda3/envs/loqus2.5/bin/loqusdb", "config_path": "/home/user/settings/loqus_default.yaml"}, # Example of an executable loqusdb binary file
+  "loqus_api" : {"api_url": "http://127.0.0.1:9000"}, # Example or a loqusdb instance reachable via REST API,
+  ..
+}
 ```
+The example above is showing the LOQUSDB_SETTINGS parameter as a dictionary containing 2 key/values, reflecting 2 eventual instances of loqusdb, one based on the executable binary file (option 1) and one reachable via REST API (option2). To reflect the infrastructure needs, it is possible to set as many connections to loqusdb instances as required.
 
-2.1 Configure Institute via Scout Browser
-Go to sidebar 'Settings'. Find 'LoqusDB id' and enter the configured id from 
-config.py. Click Save. Your configuration is now active.
-
-![Screenshot 2020-07-10 at 12 52 16](https://user-images.githubusercontent.com/1150065/87147271-9ea50600-c2ac-11ea-9f66-333b37783d52.png)
-
-URL for institute settings: `http://localhost:5000/overview/<institute>/settings`
+Please note that the key used to define each of these instances (in this case default and loqus_api) will be later used in Scout to switch among the available instances (Scout at the moment doesn't support using different loqusdb instance at the same time, and it will be only possible to use one at the time for each institute).
```

Comment:
Tough we will likely change that in the nearish future - as soon as cg supports balsamic loqusdb uploads I guess. It is correct as is, so your call to keep it, but we may have to remember to change it rather soon.

Reply:
Yes, of course, we have an open issue about this as well, but for the time being this is the actual situation and we should document it as it is I think.